data-store/chapter.md

```diff
@@ -91,88 +73,53 @@ even if you're just writing JSON to disk:
   (unlikely for most applications on modern desktop computers&hellip;but
   not unlikely for a mobile device or server-side web application)
 
-DBDB provides a simple example
-of how a database works.
-It will help inform your use of other databases
-from performance,
-crash recovery,
-and durability[^durability] perspectives.
+However, if you want to _understand_ how a database handles all of these
+problems, writing one for yourself can be a good idea. 
+
+The techniques we discuss here should also be more universally applicable to
+any problem that need to have rational, predictable behaviour when faced with 
+failure.
```

Comment:
"The techniques and concepts we discuss here should be applicable to any problem that needs to have rational, predictable behaviour when faced with failure."

?